x-pack/test_serverless/api_integration/test_suites/search/config.ts

```diff
@@ -21,5 +22,22 @@ export default createTestConfig({
   kbnServerArgs: [
     // useful for testing (also enabled in MKI QA)
     '--coreApp.allowDynamicConfigOverrides=true',
+    '--xpack.dataUsage.enabled=true',
+    '--xpack.dataUsage.autoops.api.url=http://localhost:9000',
+    `--xpack.dataUsage.autoops.api.tls.certificate=${path.resolve(
```

Comment:
i had to create some fake credentials which i put in our test directory, otherwise we'll get errors in the api that they're missing.  i'm not sure this will work in MKI.